test/src/unit-capi-fragment_info.cc

```diff
@@ -351,6 +351,17 @@ TEST_CASE(
   rc = tiledb_fragment_info_get_fragment_name(ctx, fragment_info, 1, &name);
   CHECK(rc == TILEDB_OK);
```

Comment:
Tests for deprecated functions should be either (1) isolated as a separate test and marked as testing a deprecated function or (2) removed. This is to avoid the next person needing to analyze the code from scratch to understand whether there's a dependency that needs to be removed.